recipe/meta.yaml

```
@@ -24,8 +24,8 @@ requirements:
     - wheel
   run:
     - python
-    - importlib_metadata >=3.6   # [py<310]
-    - typing_extensions >=4.4.0  # [py<311]
+    - importlib_metadata >=3.6  # [py<310]
+    - typing_extensions >=4.10.0
```

```suggestion
    - typing_extensions >=4.10.0  # [py<313]
```
https://github.com/agronholm/typeguard/blob/4.2.1/pyproject.toml#L29C52-L29C52